src/components/Card.js

```diff
@@ -13,18 +26,35 @@ class Card {
 
     this._likeIcon.addEventListener("click", () => this._handleLikeIcon());
 
-    this._trashIcon.addEventListener("click", () => this._handleDeleteCard());
+    this._trashIcon.addEventListener("click", () => {
+      this._handleDeleteClick(this._id);
+      // this._handleDeleteCard();
+    });
   }
 
   _handleLikeIcon() {
     this._likeIcon.classList.toggle("card__like-button_active");
```

Comment:
You need to **toggle likes** only in block `then` after a successful response from the server because if there is an error the user will not know about that and will see the toggled heart color.

Reply:
Corrected